share/pot/exec.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+# shellcheck disable=SC3033,SC3040,SC3043
+:
+
+exec-help()
+{
+	cat <<-"EOH"
+	pot exec [-hvdt] [-e var=value] [-u username] [-U username]
+	         -p pot COMMAND
+	  -h print this help
+	  -v verbose
+	  -d detach, run in background
+	  -t allocate pty
+	  -e var=value : set environment variable, can be repeated
+	  -u username: The username from the host environment as whom the
+	               command should run
+	  -U username: The username from the pot environment as whom the
+	               command should run
+	  -p pot : the pot image
+
+	  COMMAND is the command to execute and will be executed in the
+	  username's $HOME
+	EOH
+}
+
+# Actually send signal to process inside pot
+# $1 pot name
+# $2 detach
+# $3 env (encoded with save_params)
+# $4 alloc_pty
+# $5 user_host
+# $6 user_pot
+# $7-$n command/args
+_exec_cmd()
+{
+	local _pname _detach _env _alloc_pty _user_host _user_pot
+	local _cmd
+
+	_pname=$1; shift
+	_detach=$1; shift
+	_env=$1; shift
+	_alloc_pty=$1; shift
+	_user_host=$1; shift
+	_user_pot=$1; shift
+
```

Comment:
For consistency with the other commands, I would use `$1` through `$6` and then to a `shift 6`.
It's just style, tho.

Reply:
I’m not religious about this, but my thinking was that it might be safer this way for future maintenance - adding or removing a parameter and not changing the shift number might be an easy mistake to make - but I’m ok to change it, if consistency is more important (and the argument about maintenance could also be made the other way round, like if someone deletes a line by accident it’s less obvious). So I’ll change it to match your style once at the keyboard.

Reply:
Thanks. It's also consistent with the comment/documentation above the function.